src/types/cart.d.ts

```diff
@@ -59,45 +60,84 @@ export interface Cart extends Identifiable, CartBase {
 }
 
 export interface CartItemBase {
-  tax: {}[];
-  quantity: number;
+}
```

Comment:
What’s the purpose of the empty interface? CartItem the the only interface extending this - can we remove it?

Reply:
`...Base` interfaces are there for extending. Customers can add flow defined props there and they will bubble up and show in `CartItem` as well. I didn't want to merge them into one for consistency since other types have non-empty Base interfaces.